app/config.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
this file shouldn't be included in the merge

Reply:
it belongs just to the alpha branch

Reply:
It was a mistake. Solved in next commit